### ISSUE TITLE
Miss vdi when run "collie cluster snapshot load ..."  

### ISSUE THREAD

**Issue by gangzai** [sheepdog/sheepdog#290]
Dear All:
    I am using sheepdog daemon version 1.0 (sheepdog stable release v1.0).  
    I take two snapshots and delete the first snapshot, make a cluster snapshot and reload it. The result is not correct. 
    I tested agian. The results show that If I do not delete any snapshot of the vdi, the cluster snapshot load is correct!

My tests:
step1: collie cluster format -c 1
step2: collie vdi create test 4T
step3: collie vdi snapshot -s snapshot1 test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
s test         1  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:38   7c2b25      1     snapshot1
  test         0  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:38   7c2b26      1   
step4: collie vdi delete -s snapshot1 test
step5: collie vdi snapshot -s snapshot2 test
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
s test         2  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:38   7c2b26      1     snapshot2
  test         0  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:39   7c2b27      1     
step6: collie cluster snapshot save cluster_snapshot1 /opt/backup/
step7: collie cluster snapshot load cluster_snapshot1 /opt/backup/
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
c test         0  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:40   7c2b25      1              
s test         2  4.0 TB  0.0 MB  0.0 MB 2016-07-14 09:38   7c2b26      1     snapshot2

The clone of vdi test appears. Howerver, we miss the vdi named test!

**Comment by mitake** [comment]
@gangzai thanks for your report. However, the cluster snapshot feature is not maintained and probably we'll remove it in the feature. Because "storing every vdi in a single drive" is clearly a bad concept. If entire data can be stored in a single drive (even with some compression and dedup), distributed storage systems like sheepdog isn't required. How do you think?

**Comment by gangzai** [comment]
@mitake Thanks for your reply! 
I agree with you! The storage capacity and backup performance are the bottleneck when we make a cluster snapshot to a single drive.

**Comment by gangzai** [comment]
@mitake  Is the vdi snapshot feature maintained? I found that the original VDI disappears after some operations: vdi snapshot,  delete snapshot, vdi rollback. Only snapshots of the vdi are left behind.

**Comment by mitake** [comment]
@gangzai yes, the snapshot is maintained. Could you provide entire command line you executed?

**Comment by gangzai** [comment]
@mitake Oh, I tested randomly before. I will test it again and see if I can reproduce the problem.
